.github/workflows/e2e.yaml

```diff
@@ -18,36 +16,39 @@ jobs:
         uses: actions/checkout@v3.3.0
         with:
           submodules: recursive
-      - name: Cache container layers
-        uses: actions/cache@v3.2.6
+
+      - uses: actions/setup-go@v3
         with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-
-      - name: Setup go
-        uses: actions/setup-go@v3
+          go-version-file: "go.mod"
+          cache: true
+
+      - uses: ko-build/setup-ko@v0.6
         with:
-          go-version: "^1.19"
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2.1.0
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2.4.1
+          version: v0.13.0
+
       - name: Set up KinD
         id: kind
         run: |
           kind create cluster --image=kindest/node:v1.25.3 --config tests/e2e/kind.yaml
+
+      - name: Set up Docker Buildx
```

Comment:
Why is docker buildx still needed?

Reply:
When I tried to make `ko` load image directly to kind on mac, it failed, so just made it load image to docker daemon and then explicitly pass it to kind `kind load docker-image`. Direct load might work on runner / Linux, IDK.
We don't need buildx per say, pure docker is enough as well.

Reply:
Okay, well It looks like it works. Really nice, I will merge it